src/designer/elsa-workflows-studio/src/components/designers/tree/elsa-designer-tree/elsa-designer-tree.tsx

```diff
@@ -213,6 +217,8 @@ export class ElsaWorkflowDesigner {
     eventBus.on(EventTypes.PasteActivity, this.onPasteActivity);
     eventBus.on(EventTypes.HideModalDialog, this.onCopyPasteActivityEnabled);
     eventBus.on(EventTypes.ShowWorkflowSettings, this.onCopyPasteActivityDisabled);
+    eventBus.on(EventTypes.TestActivityMessageReceived, this.onTestActivityMessageReceived);
+    eventBus.on(EventTypes.ActivityJsonSchemaUpdated, this.onActivityJsonSchemaUpdate);
```

Comment:
All the testing-related code needs to be moved outside of the workflow designer and instead be made part of the workflow definition editor. It's OK to make the designer extensible as needed of course.